WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
![screenshot 01_07pm fev 08 2016](https://cloud.githubusercontent.com/assets/1881933/12889292/5a49ece0-ce65-11e5-95a9-4805d9ff653b.png)

The selectors look infinite to the right.

The anchor doesn't seem to have an effect, at least here.